changelog.md

```diff
@@ -2,14 +2,26 @@
 
 ## Unreleased
 
-- Add: Add nox to run tests, build docs, build package wheel/sdist(@lwasser, #395, #397)
-- Type annotation for all file in the library (@enadeau, #384, #415)
-- Remove python 3.8 support following NEP-29 (@enadeau, #416)
+## v.15
```

Comment:
```suggestion
## v1.5
```
we've been trying to be consist with our version using a lowercase v and then the version number.